packages/cashier/src/Components/Form/crypto-fiat-converter.jsx

```diff
@@ -22,6 +22,11 @@ const Timer = props => {
             setRemainingTime(initial_time);
         }
     });
+    React.useEffect(() => {
+        if (props.percentage === 0 && !props.should_percentage_reset) {
```

Comment:
@farrah-deriv Since the using of `props.` becomes more, isn't it better to destructure the props needed properties in Timer input?